doc/nvim-tree-lua.txt

```diff
@@ -822,6 +822,10 @@ longest line.
       Type: `string | number | fun(): number|string`
       Default: `-1`
 
+    *nvim-tree.view.width.include_root*
```

Comment:
Can you think of anything else that could be added in the future? I am asking because we could have an enum `nvim-tree.view.width.include` instead of multiple `include_*`.